.github/workflows/npm-release.yml

```
@@ -0,0 +1,58 @@
+        required: true
+        options:
+          - patch
+          - minor 
+          - major
+
+      app-path:
+        type: choice
+        description: Service-Name
+        required: true
+        options:
+          - sps-frontend # todo: muss angepasst werden. eigenen Ordner verwenden
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)